extensions/adobe/experience/implementations.schema.json

```diff
@@ -0,0 +1,25 @@
+{
+  "$id": "https://ns.adobe.com/xdm-extensions/experience/analytics/analytics-extensions",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Adobe Experience Cloud ExperienceEvent Implementations",
```

Comment:
Is this an implementation of an ExperienceEvent as the title indicates or rather an implementation of data collection that generates ExperienceEvents?

Reply:
It is of data collection. I will improve the description.